### PR TITLE
Update style-guide.rst

### PR DESCRIPTION
**Pull request by amilcarlucas**
Physical units should be as explicit as possible.

### CODE REVIEW

File: dev/source/docs/style-guide.rst [hunk 1]
```diff
@@ -356,6 +356,41 @@ underscores.
 
     class ap_compass { };
 
+Functions and variables
+-----------------------
+
+Functions that return a single pysical value or variables that represent a physical value should be sufixed by the physical unit.
```
Comment (hamishwillee) [inline dev/source/docs/style-guide.rst]: typo "pysical value", "suffixed"

> Functions that return a single physical value, or variables that represent a physical value, should be suffixed with the physical unit.

File: dev/source/docs/style-guide.rst [hunk 2]
```diff
@@ -356,6 +356,41 @@ underscores.
...
+    float distance_m;
+
+**Wrong:**
+
+::
+
+    uint16 get_angle() { ... };
+    float distance;
+
+Functions or variables that represent a value relative to a frame should be sufixed by the frame first, then by the physical unit if any.
```
Comment (hamishwillee) [inline dev/source/docs/style-guide.rst]: > Functions or variables that represent a value relative to a frame should be suffixed with the frame first, then with the physical unit (if any).

**Comment by hamishwillee** [comment]
See comments. I think this makes a lot of sense but needs to be reviewed by dev team.

**Comment by amilcarlucas** [comment]
Thanks Hamish. I wanted to bring this up at the dev meeting, but my ISP decided to cut my internet access 2 minutes before the dev meeting.

**Pull request merged**